scene/gui/rich_text_label.cpp

```diff
@@ -5170,33 +5170,36 @@ void RichTextLabel::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_menu"), &RichTextLabel::get_menu);
 	ClassDB::bind_method(D_METHOD("is_menu_visible"), &RichTextLabel::is_menu_visible);
 
+	ADD_PROPERTY(PropertyInfo(Variant::STRING, "text", PROPERTY_HINT_MULTILINE_TEXT), "set_text", "get_text");
 	// Note: set "bbcode_enabled" first, to avoid unnecessary "text" resets.
```

Comment:
`bbcode_enabled` should be before `text`, or it will update all caches twice.

Reply:
I had no idea the comments were related to the order the propertied themselves were added... Makes sense though. Fixed